NohanaImagePicker/AssetCell.swift

```diff
@@ -32,6 +32,10 @@ class AssetCell: UICollectionViewCell {
 
             pickButton.setImage(droppedImage, for: UIControlState())
             pickButton.setImage(pickedImage, for: .selected)
+			
+			if let selectionTint = nohanaImagePickerController.config.color.selectionTint {
+				overlayView.backgroundColor = selectionTint
+			}
```

Comment:
spaces vs tabs 😂 the rest of the file uses spaces.... 😬

Reply:
Yeah I think once we get this shipped, we should get this repo to be all in tabs.

I didn't want to bother matching it to spaces right now.